research-hub-web/src/app/components/articles/articles.component.spec.ts

```diff
@@ -38,6 +40,233 @@ describe('ArticlesComponent', () => {
+                  'nodeType': 'list-item'
+                }
+              ],
+              'nodeType': 'unordered-list'
+            },
+            {
+              'data': {},
+              'content': [
+                {
+                  'data': {},
+                  'marks': [],
+                  'value': 'An Even Smaller Heading',
+                  'nodeType': 'text'
+                }
+              ],
+              'nodeType': 'heading-4'
+            },
+            {
+              'data': {},
+              'content': [
+                {
+                  'data': {},
+                  'marks': [],
+                  'value': 'Last thing I\'ll check is how a link works, have you heard of ',
+                  'nodeType': 'text'
+                },
+                {
+                  'data': {
+                    'uri': 'https://google.com/'
+                  },
+                  'content': [
+                    {
+                      'data': {},
+                      'marks': [],
+                      'value': 'Google',
+                      'nodeType': 'text'
+                    }
+                  ],
+                  'nodeType': 'hyperlink'
+                },
+                {
+                  'data': {},
+                  'marks': [],
+                  'value': '?',
+                  'nodeType': 'text'
+                }
+              ],
+              'nodeType': 'paragraph'
+            },
+            {
+              'data': {},
+              'content': [
+                {
+                  'data': {},
+                  'marks': [],
+                  'value': '',
+                  'nodeType': 'text'
+                }
+              ],
+              'nodeType': 'paragraph'
+            }
+          ],
+          'nodeType': 'document'
+        },
+        '__typename': 'ArticleBody'
+      },
+      'keywords': [
+        'article',
+        'informational',
+        'general',
+        'test'
+      ],
+      'slug': 'first-article',
+      'searchable': true,
+      'icon': {
+        'title': 'Pippy',
+        'description': 'Sam\'s dog Pippy',
+        'url': 'https://images.ctfassets.net/vbuxn5csp0ik/014fpqFg0KmnP8NWc9Jyxe/0a1a14a6a0dd438443f70d5f917568fc/Screen_Shot_2020-07-23_at_3.12.46_PM.png',
+        '__typename': 'Asset'
+      },
+      'relatedItemsCollection': {
+        'items': [
+          {
+            'title': 'Super dooper research service',
+            'summary': 'The best service ever. It saved my life.',
+            'slug': 'super-dooper-research-service',
+            '__typename': 'Service'
+          },
+          {
+            'title': 'Death Star',
+            'summary': 'Mobile space station and galactic superweapon.',
+            'slug': 'death-star',
+            '__typename': 'Equipment'
+          }
+        ],
+        '__typename': 'ArticleRelatedItemsCollection'
+      },
+      '__typename': 'Article'
+    } as unknown as Article);
```

Comment:
Interesting typecasting?

Reply:
It wasn't letting me cast directly to article and suggested using "as any" to fix it. I did spend time trying to have it typecast without that but I couldn't seem to get it to work and casting like that worked for the unit tests.

Reply:
As we are looking at updating pure Apollo Client service later in this sprint, we can resolve this issue after we decide on what the graphql service looks like.